ui-admin/src/HomePage.tsx

```diff
@@ -57,7 +57,7 @@ function HomePage() {
       <div className="d-flex justify-content-center mt-3">
         <div className="alert alert-warning" role="alert">
             You do not have access to any portals or studies. If this is an error, please
-            contact <a href="mailto:support@juniper.terra.bio">support@juniper.terra.bio</a>
+            contact <a href={`mailto:${SUPPORT_EMAIL_ADDRESS}`}>{SUPPORT_EMAIL_ADDRESS}</a>
```

Comment:
we've talked about doing this, but since our support address will probably change soon once we migrate away from the terra.bio domain, now seemed like a good time to quickly do it